plugins/hls-eval-plugin/src/Ide/Plugin/Eval/Parse/Token.hs

```diff
@@ -145,6 +145,8 @@ tokens = concatMap (\(l, vs) -> map (Located l) vs) . zip [0 ..] . reverse . snd
       Right (st', tokens') -> (st', tokens' : tokens)
       Left err -> error $ unwords ["Tokens.next failed to parse", ln, err]
 
+{- HLINT ignore tokens "Use zipFrom" -}
+
 -- | Parse a line of input
 aline :: State -> TParser
 aline InCode          = optionStart <|> multi <|> singleOpen <|> codeLine
```

Comment:
Line 142.

I don't think this is an improvement but we can do whatever.

plugins/hls-eval-plugin/src/Ide/Plugin/Eval/Parse/Token.hs:142:55-64: Warning: Use zipFrom
Found:
  zip [0 .. ]
Perhaps:
  zipFrom 0